debian/control

```diff
@@ -18,3 +18,27 @@ Suggests: python-oq-libs-extra (>= 1.5.0~)
 Description: computes seismic hazard and physical risk
  based on the hazard and risk libraries developed by the GEM foundation
  (http://www.globalquakemodel.org/)
+
+Package: python-oq-engine-master
+Architecture: all
+Depends: ${shlibs:Depends}, ${misc:Depends}, python-oq-libs (>=1.5.0~), rabbitmq-server (>=2.8.0-2~gem02), supervisor (>=3.0b2-1~), python-oq-engine (= ${binary:Version})
```

Comment:
I would probably remove `python-oq-libs` since `master` depends on `engine` which depends already on `libs`. The same is for `supervisor`. Thsi package does not include any service.